src/assets/top-section-graphic.svg

```diff
@@ -0,0 +1,42 @@
+<svg width="782" height="832" viewBox="0 0 782 832" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we put this in a folder? Let's say ```src/assets/home/top-section-graphic.svg```. We will have a lot of svgs and images and it's easier to group by page